content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -15,68 +15,85 @@ further_reading:
       tag: 'GitHub'
       text: 'Examples of Custom Instrumentation'
 ---
+<div class="alert alert-warning">
+  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution within your application environment.</li><li> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.</li></ul>
+</div>
 
 ## Compatibility
 
 - The .NET Tracer supports all .NET-based languages (for example, C#, F#, Visual Basic).
 
-- The .NET Tracer supports instrumentation on:
-  - .NET 5
-  - .NET Core 3.1
-  - .NET Core 2.1
+- The .NET Tracer library for Datadog is open-source. For more information, see the [tracer Github repository][1].
 
-- .NET Tracer supports the following processor architectures
-  - Windows x86 (`win-x86`)
-  - Windows x64 (`win-x64`)
-  - Linux x64 (`linux-x64`)
-  - Alpine Linux x64 (`linux-musl-x64`)
-  - Linux ARM64 (`linux-arm64`) Added in version 1.27.0, automatic instrumentation only supported on .NET 5.
+### Supported .NET Core runtimes
+The .NET Tracer supports instrumentation on .NET Core 2.1 and above. It also supports [.NET Framework][2].
 
-- The .NET Tracer library for Datadog is open-source. For more information see the [tracer Github repository][1].
+| Version              | Vendor End of Life |
+| -------------------- | ------------------ |
+| .NET 5               |                    |
+| .NET Core 3.1 (LTS)  | 12/03/2022         |
+| .NET Core 3.0        | 03/03/2020         |
+| .NET Core 2.2        | 12/23/2019         |
```

Comment:
This now makes it look like we support 2.2 and 3.0, which we don't.

Also, if this page is about what _Datadog_ supports, why are we adding the dates when Microsoft drops support for any runtime version so prominently? Is this helpful for users or just noise?

Reply:
I am happy to change the vendor to Microsoft end of life. I do think there is value here to show users that these versions have reached EOL by Microsoft.

Reply:
Updated vendor to mean Microsoft in https://github.com/DataDog/documentation/pull/11200/commits/77793e5a94b644a2679fb90ab651acd9a154876b 

I still see the value of having EOL from Microsoft shared with our customers. If we decide later its not useful then we can remove it.